.github/workflows/tekton_bundle_push.yaml

```diff
@@ -17,21 +15,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: debianmaster/actions-k3s@master
-        id: k3s
-        with:
-          version: 'latest'
+      - run: |
+          mkdir -vp $HOME/{bin,.kube} || true
+          pushd $HOME/bin
+          curl -LO https://dl.k8s.io/release/v1.23.0/bin/linux/amd64/kubectl
+          popd
+          PATH="$HOME/bin:$PATH"
```

Comment:
Thanks, yeah having curl drop the file to the destination is better. But it's a moot point, I've removed these lines because apparently `kubectl` is not required. I tested, and it works without `kubectl`, so I guess `tkn` was looking for the `~/.kube/config` itself instead of wrapping `kubectl`. My bad.

Reply:
fixed